### ISSUE TITLE
Release a new version

### ISSUE THREAD

**Issue by mlhamel** [steve-m/librtlsdr#35]
Since most packaging system are pointing to official release (Example: homebrew is pointing to 0.5.3).

But right now, 0.5.3 is almost 3 years old.

That would be awesome if someone could release 0.5.4, ideally having a release that would include support for `rtlsdr_set_tuner_bandwidth`.

thanks

**Comment by Romeo-Golf** [comment]
I agree that a new release is long overdue, perhaps the relatively few changes which been accepted since 0.5.3 was released in February 2014 could be rolled up into a 0.5.4 release?

**Comment by bogdanr** [comment]
I second that. We can't build other things that depend on this lib if we use the 0.5.3 version.
